ui/src/app/components/apps-panel/expanded-app.spec.tsx

```diff
@@ -220,59 +219,10 @@ describe('ExpandedApp', () => {
 
   const gkeAppTypes = [UIAppType.CROMWELL, UIAppType.RSTUDIO];
   describe.each(gkeAppTypes)('GKE App %s', (appType) => {
-    it('should allow clicking Pause when the app status is RUNNING', async () => {
```

Comment:
I considered commenting these out for when we re-add pause/resume. However, I'd prefer to not leave dead code around. Instead, [I left a comment](https://precisionmedicineinitiative.atlassian.net/browse/RW-9623?focusedCommentId=155078) on the relevant bug so we can copy these back as needed.